test/e2e_node/pods_lifecycle_termination_test.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	PodSigkillTimout = 5 * time.Minute
+)
+
+// Pod sigkill test will cover pods with graceful termination period set but failed
+// to terminate and forcefully killed by kubelet. This test exaimine pod's container's
+// exit code is 137 and the exit reason is `Error`
+var _ = SIGDescribe("Pod sigkill [LinuxOnly] [NodeConformance]", func() {
+	f := framework.NewDefaultFramework("sigkill-test")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
```

Comment:
This should not be needed since the container is not doing anything that should require privileged.

Reply:
I still don't fully understand that because the pod itself is not doing any privileged operation so why should it require privileged? 

Have you tried `f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline`